src/lib/components/Websocket.svelte

```diff
@@ -1,33 +1,28 @@
 <script lang="ts">
 	import { socket } from '$lib/stores/stateStore';
-	import { svgs, type Svg } from '$lib/stores/svgStore';
+	import { svgs } from '$lib/stores/svgStore';
 	import { onMount } from 'svelte';
+	import type { Edit, Remove, Svg } from '$lib/types';
 
 	onMount(() => {
 		$socket.on('edit', handleEdit);
 		$socket.on('remove', handleRemove);
 	});
 
 	// TODO: Mads, optionals i interface.
-	function handleEdit(data: any) {
+
+	function handleEdit(data: Edit) {
 		// check if the command already exists
 		const commandIndex = $svgs.findIndex((svg: Svg) => svg.commandId === data.commandId);
 
 		// if it exists, modify it
-		// HACK: Maybe rewrite to loop through the keys of data instead
-		if (commandIndex !== -1) {
-			// only modify the property if it exists on the data object
-			if ('x' in data) {
-				$svgs[commandIndex].x = data.x;
-			}
-			if ('y' in data) {
-				$svgs[commandIndex].y = data.y;
-			}
-			if ('svg' in data) {
-				$svgs[commandIndex].svg = data.svg;
-			}
+		if (commandIndex === -1) {
```

Comment:
Jeg er ret sikker på at det her er en fejl. Du har omvendt denne her condition, så den er "===" i stedet for "!=="